pkgs/applications/office/ledger/default.nix

```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  cmakeFlags = [ "-DCMAKE_INSTALL_LIBDIR=lib" ];
+  cmakeFlags = [ "-DCMAKE_INSTALL_LIBDIR=lib" "-DUSE_PYTHON=true" ];
```

Comment:
Can we turn this into a package option?

Reply:
Sadly, I wouldn't know how.

Reply:
You'd add a parameter to the input of the function which is true by default: `{... usePython ? true ... }:`

then you'd check it in the `cmakeFlags` like this: `(if usePython then "-DUSE_PYTHON=true" else "");` or you'd use `lib.optionalString usePython "-DUSE_PYTHON=true"`.

Reply:
Oh, easy enough. I'll update the PR.

Reply:
Done